### ISSUE TITLE
Finetuning on custom spanish dataset

### ISSUE THREAD

**Issue by anirindg** [dccuchile/beto#19]
I have a custom spanish dataset which I want to classify into specific labels. However, in the repo, there are no specific instructions ( apart from the weight files ), on how to do that ( specifically, using TF2 ). Can you be more detailed ?

**Comment by Smolky** [comment]
Hello. In case it helps, I address this problem fine-tunning the model using HuggingFace
https://huggingface.co/transformers/training.html

Particularly, using the Trainer class is quite straightforward.

**Comment by josecannete** [comment]
Hello @anirindg, thank for your interest in the project.

The documentation of the [Transformers library from HuggingFace](https://github.com/huggingface/transformers) is really good, with lots of examples. I encourage you to explore it.

The link provided by @Smolky is probably what you need. This other one might be helpful too: https://huggingface.co/transformers/custom_datasets.html

I hope it helps!
